### DISCUSSION TITLE
Help and Troubleshooting

### DISCUSSION THREAD

**Discussion by riffnshred** [riffnshred/nhl-led-scoreboard#218] [Announcements]
From this moment, help and troubleshooting will be provided through the [Issues section](https://github.com/riffnshred/nhl-led-scoreboard/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc).

This is so we can keep track of all the issues AND solutions so that anyone facing a problem can find an answer quickly. The alternative result in the information being spread across platforms and or lost. The Issue section has a search bar and filters to help you find anything using keywords.

**Comment by rhazzed** [comment]
Riff...  can you elaborate on the --terminal-mode=true option?    once enabled how does one use it?

**Reply by rhazzed** [reply]
Nevermind.. figured it out.  that's so cool!

**Reply by riffnshred** [reply]
That is a feature where the scoreboard will appear in the terminal. That was develop by @ELepolt

**Reply by rhazzed** [reply]
now I need an ssh client for my wstch...

**Reply by MordeKyle** [reply]
rhazzed where is this option and how did you get it working?

**Reply by rhazzed** [reply]
1. ssh in to your scoreboard 
2. cd in to your install directory (cd nhl-led-scoreboard)
3. type:  python3 src/main.py --terminal-mode=true

If your ssh/terminal window supports terminal emulation, it should start displaying on your screen.  Looks like a great way to debug things.

**Reply by MordeKyle** [reply]
Awesome! This is very helpful! Thanks!

**Comment by rhazzed** [comment]
![20211012_111720](https://user-images.githubusercontent.com/25889638/137001164-a535c772-db4c-4a12-997c-cf8c93c45f89.jpg)
